server/re/src/printMS.cpp

```diff
@@ -173,7 +173,9 @@ int writeBytesBuf( msParam_t* where, msParam_t* inBuf, ruleExecInfo_t *rei ) {
     if ( inBuf->inpOutBuf ) {
         /* Buffer might no be null-terminated */
         writeStr = ( char* )malloc( inBuf->inpOutBuf->len + 1 );
-        strncpy( writeStr, ( char* )inBuf->inpOutBuf->buf, inBuf->inpOutBuf->len );
+        if (nullptr != inBuf->inpOutBuf->buf) {
+            strncpy(writeStr, (char*) inBuf->inpOutBuf->buf, inBuf->inpOutBuf->len);
+        }
         writeStr[inBuf->inpOutBuf->len] = '\0';
```

Comment:
Same thing here - wondering whether we should put `\0` in `writeStr[0]` in the `else` case to avoid a garbage string.

Reply:
Done.